xarray/core/computation.py

```diff
@@ -801,7 +822,7 @@ def apply_ufunc(
     join: str = "exact",
     dataset_join: str = "exact",
     dataset_fill_value: object = _NO_FILL_VALUE,
-    keep_attrs: bool = False,
+    keep_attrs: Union[bool, str, None] = False,
```

Comment:
should the default value become `None` (falling back to `False` in the `_get_keep_attrs` call)?